src/UnityExtension/Assets/Editor/GitHub.Unity/UI/Window.cs

```diff
@@ -24,8 +25,10 @@ class Window : BaseWindow
         private const string Window_RepoBranchTooltip = "Active branch";
 
         [NonSerialized] private double notificationClearTime = -1;
+        [SerializeField] private SubTab? nextTab;
```

Comment:
SubTab is an enum so there's no reason to use a nullable here, just add a None option to it to know what there is no current selection.